src/features/expense/ui/tests/MonthSelector.test.tsx

```
@@ -0,0 +1,8 @@
+import { describe, it } from 'vitest';
+
+describe('MonthSelector', () => {
+  it('renders previous button.', () => {});
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)